src/reducers/attributes/index.js

```diff
@@ -0,0 +1,44 @@
+import {
+    LOADING_ATTRIBUTES_ERROR,
+    LOADING_ATTRIBUTES_START,
+    SET_ATTRIBUTES,
+} from './actions'
+
+const initialState = {
+    loading: false,
+    error: '',
+
+    dataElement: [],
+    organisationUnit: [],
+}
+
+export default function attributesReducer(
+    state = initialState,
+    { type, payload } = {}
+) {
+    if (type === LOADING_ATTRIBUTES_START) {
```

Comment:
This is also nice actually, vs the standard switches in reducers.

Reply:
yeah, the app already does it this way, but I also started using this approach.
I like to take `lodash/fp`s `isEqual function and do this in my personal projects:

```js
import { isEqual } from 'lodash/fp'

const reducer = (state, { type, payload }) => {
  const isType = isEqual(type)

  if (isType(ACTION_TYPE)) {
    return ...
  }

  return state
}
```